Havit/Havit/Screens/CategoryContents/View/CategoryContentsViewController.swift

```diff
@@ -6,24 +6,184 @@
+            make.trailing.equalTo(filterView).offset(-16)
+            make.width.equalTo(47)
+            make.height.equalTo(15)
+        }
+        
+        filterCollectionView.snp.makeConstraints { (make) -> Void in
+            make.leading.equalTo(filterView).offset(0)
+            make.bottom.equalTo(filterView).offset(-9)
+            make.width.equalTo(250)
+            make.height.equalTo(31)
+        }
+        
+        contentsCollectionView.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(filterView).offset(67)
+            make.leading.equalTo(mainView).offset(0)
+            make.trailing.equalTo(mainView).offset(0)
+            make.bottom.equalTo(mainView).offset(0)
+        }
+    }
+}
+
 
-    /*
-    // MARK: - Navigation
+// MARK: - Extensions
 
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+extension CategoryContentsViewController: UISearchBarDelegate {
+    
+}
+
+extension CategoryContentsViewController: UICollectionViewDelegate {
+    
+}
+
+extension CategoryContentsViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        switch collectionView {
+        case filterCollectionView:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: CategoryFilterCollectionViewCell.cellID, for: indexPath) as! CategoryFilterCollectionViewCell
+            cell.filterNameLabel.text = "앙대"
+            
+            return cell
+        case contentsCollectionView:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ContentsCollectionViewCell_sort2.cellID, for: indexPath) as! ContentsCollectionViewCell_sort2
+            cell.backgroundColor = .white
+            
+            return cell
+        default:
+            return UICollectionViewCell()
+        }
     }
-    */
+}
 
+extension CategoryContentsViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        switch collectionView {
+        case filterCollectionView:
+            return 3
+        case contentsCollectionView:
+            return 10
+        default:
+            return 0
+        }
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        switch collectionView {
+        case filterCollectionView:
+            return CGSize(width: 50, height: 31)
+        case contentsCollectionView:
+            return CGSize(width: view.frame.width / 2 - 9, height: 139)
```

Comment:
p3;
괄호를 사용해 우선적으로 계산해줘야 할 친구들을 표기해주면 연산자 우선순위를 따로 생각해주지 않아도 되어 더 보기 좋을 것 같습니다!
```suggestion
            return CGSize(width: (view.frame.width / 2) - 9, height: 139)
```

Reply:
세심한 코리 감사합니다 😃  반영하도록 하겠습니다!